iceoryx_posh/source/roudi/roudi_process.cpp

```diff
@@ -581,18 +581,30 @@ void ProcessManager::addNodeForProcess(const ProcessName_t& processName, const N
     RouDiProcess* process = getProcessFromList(processName);
     if (nullptr != process)
     {
-        runtime::NodeData* node = m_portManager.acquireNodeData(processName, nodeName);
+        m_portManager.acquireNodeData(processName, nodeName)
```

Comment:
For now I would keep it. I have no other idea how to setup a multi-process database with available nodes for rmw_iceoryx. We can discuss this once we touch rmw_iceoryx again

Reply:
We do not use ``NodeData`` at the moment but it could be useful if we want to store additional information (like a device id which could be necessary if we will have GPU support).